### DISCUSSION TITLE
more dGPU overclock offset?

### DISCUSSION THREAD

**Discussion by yueh-hsuan** [seerge/g-helper#1937] [Q&A] [answered]
Excuse me. My model is ROG G14 2023 RTX 4060, and I can overclock the GPU frequency to +300mhz/+1000mhz in MSI Afterburner, but the upper limit of G-helper can only reach +250mhz/+500mhz. Is there any adjustment range in the software? Thanks.

**Comment by seerge** [comment, marked as answer]
@yueh-hsuan hello, it's already possible

https://github.com/seerge/g-helper/wiki/Power-user-settings#higher-maximum-and-minimum-gpu-clock--memory-offsets

**Reply by yueh-hsuan** [reply]
ok Thanks.
